content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst

```diff
@@ -2,60 +2,117 @@
+  Used to correct stock levels of physical inventory.
+- :guilabel:`Production`: virtual counterpart location for production operations. This location
+  consumes components and produces finished products.
+- :guilabel:`Transit Location`: counterpart location that should be used in inter-company or
+  inter-warehouse operations.
+
+.. image:: use_locations/use-locations-new-location-form.png
+   :align: center
+   :alt: Additional Information section of new location creation form.
+
+Configure the remaining fields in the :guilabel:`Additional Information` section as follows:
+
+- :guilabel:`Company`: the company whose warehouse the location is inside of. Leave this field blank
+  if this location is shared between companies.
+- :guilabel:`Is a Scrap Location?`: tick this checkbox to allow for scrapped/damaged goods to be
+  stored in this location.
+- :guilabel:`Is a Return Location?`: tick this checkbox to allow products to be returned to this
+  location.
+- :guilabel:`Barcode`: the barcode assigned to the location.
+- :guilabel:`Replenish Location`: tick this checkbox to get all quantities to replenish at this
+  location.
+
+In the :guilabel:`Cyclic Counting` section, change the value in the :guilabel:`Inventory Frequency
+(Days)` field from the default `0`, if necessary.
+
+.. image:: use_locations/use-locations-cyclic-counting.png
    :align: center
-   :alt: The form for creating a new location.
+   :alt: Cyclic Counting section of new location creation form.
+
+When different than `0`, the inventory count dates for products stored at this location are
+automatically set at the defined frequency.
+
+In the :guilabel:`Logistics` section, in the :guilabel:`Removal Strategy` field, click the drop-down
+menu and select the :doc:`removal strategy <../removal_strategies>` for how items should be removed
+from this location.
 
 .. _inventory/location-hierarchy:
 
 Create location hierarchies
 ===========================
 
 The *Parent Location* setting on the new location form allows for a location to exist within a
-warehouse or another location. Every location can serve as a parent location, and every parent
-location can have multiple locations within it, allowing for the creation of a virtually infinite
-hierarchical structure.
+warehouse, or below another location.
+
+Every location can serve as a parent location, and every parent location can have multiple locations
+within it, allowing for the creation of a hierarchical structure.
 
 .. example::
-   Location hierarchy could be organized so that a shelf is located within an aisle, which is
-   located within a room, which is located within the overall warehouse.
+   Location hierarchy could be organized so that a small refrigerator is located on a specific
+   shelf, which is located in the warehouse.
+
+   .. image:: use_locations/use-locations-hierarchy.png
+     :align: center
+     :alt: Location hierarchy of small refrigerator, on a shelf in a warehouse.
```

Comment:
Small fix - indent the image attributes by one more space :)
```suggestion
   .. image:: use_locations/use-locations-hierarchy.png
      :align: center
      :alt: Location hierarchy of small refrigerator, on a shelf in a warehouse.
```

Reply:
Nice catch - suggestion implemented ✅